### PR TITLE
[fix](cloud) add cloud cluster info for `admin show replica distribution from`

### PR DESCRIPTION
**Pull request by deardeng**
## Proposed changes

Issue Number: close #xxx

before：
```
mysql> admin show replica distribution from cloud_cache_hotspot;
+-----------+------------+-------------+-----------------------------------+------------+-----------+-------------+
| BackendId | ReplicaNum | ReplicaSize | NumGraph                          | NumPercent | SizeGraph | SizePercent |
+-----------+------------+-------------+-----------------------------------+------------+-----------+-------------+
| 10002     | 37         | 0           | >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> | 33.64 %    |           | 0.00%       |
| 10003     | 37         | 0           | >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> | 33.64 %    |           | 0.00%       |
| 10004     | 36         | 0           | >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>  | 32.73 %    |           | 0.00%       |
+-----------+------------+-------------+-----------------------------------+------------+-----------+-------------+
```

after
```
mysql> admin show replica distribution from cloud_cache_hotspot;
+-----------+------------+-------------+-----------------------------------+------------+-----------+-------------+------------------+--------------------+
| BackendId | ReplicaNum | ReplicaSize | NumGraph                          | NumPercent | SizeGraph | SizePercent | CloudClusterName | CloudClusterId     |
+-----------+------------+-------------+-----------------------------------+------------+-----------+-------------+------------------+--------------------+
| 10002     | 36         | 0           | >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>  | 32.73 %    |           | 0.00%       | compute_cluster  | compute_cluster_id |
| 10003     | 37         | 0           | >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> | 33.64 %    |           | 0.00%       | compute_cluster  | compute_cluster_id |
| 10004     | 37         | 0           | >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> | 33.64 %    |           | 0.00%       | compute_cluster  | compute_cluster_id |
+-----------+------------+-------------+-----------------------------------+------------+-----------+-------------+------------------+--------------------+

```

### CODE REVIEW

**Comment by deardeng** [comment]
run buildall

**Comment by gavinchou** [comment]
Add an example to show the output (before and after this change)

**Comment by deardeng** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowReplicaDistributionStmt.java [hunk 1]
```diff
@@ -37,6 +37,7 @@ public class ShowReplicaDistributionStmt extends ShowStmt {
             .add("BackendId").add("ReplicaNum").add("ReplicaSize")
             .add("NumGraph").add("NumPercent")
             .add("SizeGraph").add("SizePercent")
+            .add("Cloud").add("CloudClusterName").add("CloudClusterId")
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowReplicaDistributionStmt.java:40]: cloud column seems redundant

Comment (deardeng) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowReplicaDistributionStmt.java:40]: 多一个不多

**Comment by deardeng** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Pull request merged**